### ISSUE TITLE
test some games depending on wmp10

### ISSUE THREAD

**Issue by frostworx** [sonic2kk/steamtinkerlaunch#38]
testing if support for some wmp10 games could be implemented:
search proton issues and found those I own:

https://github.com/ValveSoftware/Proton/issues/1649   :  looks like wmp10 and quartz are not enough: #39

https://github.com/ValveSoftware/Proton/issues/829     : nope, neither wmp10+quartz, nor wmp11+quartz(+ disabled winegstreamer) fixed this glitch

https://github.com/ValveSoftware/Proton/issues/770     : the game start and seems to be very playable with HACKWMP10=1 and 
WINETRICKSPAKS=d3dx9_43 but videos are black (cinepak,directshow,quartz didn't help)

https://github.com/ValveSoftware/Proton/issues/576     : damn, both titles already do work with wmp11, adding both as tweaks

Would be nice if you test other titles you own.
With some luck a tweakfile `($SteamAppId.conf)` with 
`HACKWMP10=1`
is already enough

**Comment by frostworx** [comment]
also testing
https://github.com/ValveSoftware/Proton/issues/4052
edit: nope - this game simply has CEG protection...

**Comment by frostworx** [comment]
I have the feeling that the wmp10 hack doesn't make games crash anymore, but also doesn't help alone to get videos working
(at least also the both Giana titles have a short black screen at the beginning, this could be a missing logo).
Haven't debugged this yet, so no new issue for now (any maybe even not at all...)
edit:
Not worth the effort, for the few games.
Both Giana games do work fine even if videos are missing, which probably are the two most important wmp10 games.
Pull requests unlikely but welcome
